.devcontainer/devcontainer.json

```diff
@@ -7,23 +7,22 @@
 	"customizations": {
 		"vscode": {
 			"extensions": [
-				"dbaeumer.vscode-eslint"
+				"dbaeumer.vscode-eslint",
+				"mhutchie.git-graph",
```

Comment:
Could you please give me some clues about why those lines are added?

Reply:
I'm adding a few vscode extensions for people that want to work on the project using github codespace:
- eslint and typescript
- jest to run the utests more easily
- gitgraph to better view the git history
- markdown enhanced to have a markdown preview that match the custom markdown renderer of github

Technically, those extensions can be added by the user regardless, or part of there own vscode profile. But when using codespace, some extension prefer to run "in the codespace", so it's faster to register them directly in the devcontainer.json